builtins/amp-video.js

```diff
@@ -98,6 +121,80 @@ export function installVideo(win) {
     isVideoSupported_() {
       return !!this.video_.play;
     }
+
+    /**
```

Comment:
No jsdoc here. Instead you can add this info to each method as `/** @override See VideoInterface */` or such.